src/test/resources/epub3/content-publication.feature

```diff
@@ -198,6 +198,9 @@ Feature: EPUB 3 ▸ Content Documents ▸ Full Publication Checks
     Then error OPF-013 is reported
     And no other errors or warnings are reported
 
+  #####  script
+  
+  #//TODO verify script core media types
```

Comment:
Are you thinking of adding tests on the media type for script src references here?

(I don't think it matters, spec-wise, what you use in the content so long as one of the three JS types is declared is declared in the manifest, but maybe could be a usage note.)

Reply:
Or are you thinking of putting text/vbscript or some junk in to make sure it's checked to be a CMT?

Reply:
the idea was to add tests for other HTML-allowed Javascript MIME types, like `application/x-javascript`, `text/ecmascript`, `text/jscript`, etc…

If the script is declared as such in the manifest, a warning/error should be raised that it is not a CMT.